.github/workflows/basic.yml

```diff
@@ -204,28 +204,39 @@ jobs:
         pool_tracking: ['ON', 'OFF']
         shared_library: ['OFF']
         sanitizers: [{asan: OFF}]
+        os_provider: ['ON']
         include:
           - os: 'windows-2022'
             build_type: Release
             compiler: {c: clang-cl, cxx: clang-cl}
             pool_tracking: 'ON'
+            os_provider: ['ON']
```

Comment:
`['ON']` -> `'ON'` 😉

Reply:
Done.

Reply:
not quite - pls only update in `include`s, not in `matrix` (a list is required there)

Reply:
Ok, done.